awscli/examples/iot-data/update-thing-shadow.rst

```diff
@@ -2,13 +2,40 @@
 
 The following ``get-thing-shadow`` example gets the current state of the device shadow for the thing named ``MyRPi`` and saves it to the file ``output.txt``. ::
```

Comment:
I think `get-thing-shadow` at the start of the sentence here needs to be updated to `update-thing-shadow`